bitrise.yml

```diff
@@ -0,0 +1,71 @@
+format_version: 7
+default_step_lib_source: https://github.com/bitrise-io/bitrise-steplib.git
+
+app:
+  envs:
+  - GOOGLE_SERVICE_ACCOUNT: $GOOGLE_SERVICE_ACCOUNT
+  - FLANK_CONFIG: $FLANK_CONFIG
+  - SAMPLE_ANDROID_APP_GIT_URL: "https://github.com/bitrise-samples/android-multiple-test-results-sample"
+
+workflows:
+  test:
```

Comment:
Could you wire in an iOS test as well? If the step supports iOS too [(iOS support in step.yml)](https://github.com/bitrise-steplib/bitrise-step-flank/pull/1/files#diff-91d8d269a1c9bd3631ddb04b80b01c4eR12) , then I think it should test it too.